website/templates/profile/account.mako

```diff
@@ -5,39 +5,101 @@
 
 <%def name="content()">
     <% from website import settings %>
-    <h2 class="page-header">Account Settings</h2>
-    <div class="row">
-        <div class="col-md-3">
-            <div class="panel panel-default">
-                <ul class="nav nav-stacked nav-pills">
-                    <li><a href="${ web_url_for('user_profile') }">Profile Information</a></li>
-                    <li><a href="#">Account Settings</a></li>
-                    <li><a href="${ web_url_for('user_addons') }">Configure Add-ons</a></li>
-                    <li><a href="${ web_url_for('user_notifications') }">Notifications</a></li>
-                </ul>
+    <div id="accountSettings">
```

Comment:
You can add the `scripted` CSS class to hide this div until the view model is bound; this prevents a Flash of Unstyled Content.